anvil/server/src/lib.rs

```diff
@@ -34,6 +39,14 @@ where
         .layer(Extension(http))
         .layer(Extension(ws))
         .layer(TraceLayer::new_for_http())
+        .layer(
+            // see https://docs.rs/tower-http/latest/tower_http/cors/index.html
+            // for more details
+            CorsLayer::new()
+                .allow_origin("*".parse::<HeaderValue>().unwrap())
```

Comment:
Should we make this configurable via CLI and default to `*`?

Reply:
sg, will introduce a builder type for all things network/server